src/Microsoft.Health.Fhir.Shared.Core/Features/Resources/Create/CreateResourceHandler.cs

```diff
@@ -38,6 +44,8 @@ public async Task<UpsertResourceResponse> Handle(CreateResourceRequest message,
             // If an Id is supplied on create it should be removed/ignored
             resource.Id = null;
 
+            await ResolveBundleReferencesAsync(resource, _referenceIdDictionary, resource.ResourceType.ToString(), cancellationToken);
```

Comment:
Are we only resolving conditional reference here? What would be the behavior in non-conditionalreference such as  resource/resourceid or any urn?

Reply:
Also we might want to change the name of methodname as we are no longer resolving only bundle reference?